go-app-ussd_nurse_rapidpro.js

```diff
@@ -150,23 +150,20 @@ go.app = function() {
                 question: ("This number previously opted out of NurseConnect messages. Are <you/they> sure <you/they> want to sign up again?"),
                 choices: [
                     new Choice('yes', ('Yes')),
-                    new Choice('no', ('No'))
+                    new Choice('state_no_subscription', ('No'))
                 ],
                 next: function(choice) {
                     if (choice.value === 'yes') {
-                        var optin_info = {
-                            "request_source": self.im.config.name || "ussd_nurse",
-                            "address_type": "msisdn"
+
                             //identity will be set here
                             //address will be set here
-                        };
-                        return is
-                        .optin(optin_info)
-                        .then(function() {
-                            return 'state_faccode';
-                        });
-                    }else {
-                        return self.states.create('state_no_subscription');
+                            //address type will be set here
```

Comment:
wouldn't block comments be better

Reply:
These all will be replaced once new functions that Seed functions, but will change 👍